### ISSUE TITLE
Redémarrge PC après session meterpreter/Pc reboot after meterpreter session 

### ISSUE THREAD

**Issue by STE4LTH** [ElevenPaths/Eternalblue-Doublepulsar-Metasploit#36]
Bonjour, après avoir lancé le module sur l'ordinateur de la victime, je reçois une session meterpreter, mais quelques secondes/minutes après l'ordinateur redémarre.
Si quelqu'un sait quoi faire qu'il se montre ! ;-) 
Merci d'avance ! (victime : architecture x64, processus à injecter : lsass.exe)

Hello, I'm french but if you want I can speak in english if somebody wants. After have send the module on the victim's computer, I get meterpreter session, but few seconds/minutes after the computer reboot.
If someone know what I can do, show yourself ! ;-) 
Thank you in advance ! (victim : architecture x64, process inject : lsass.exe)

**Comment by nithilin** [comment]
i have same problem

**Comment by UrielRicardo** [comment]
@STE4LTH @nithilin  
Use the "migrate" command from the "meterpreter" tool, when establishing connection with your victim will be using a process, as in the case of a target windows 2008 r2, x64 the process used is lsass.exe, using only one process is And can terminate the connection, causing the machine to restart.
The migrate command allows you to change the process by which you are holding the connection.

example:https://www.youtube.com/watch?v=4VbgCQc-m2g

**Comment by STE4LTH** [comment]
Thank you, I know this command, but the process lsass is interesting because is on "authority\system". And I tried it again and sometimes the computer restart without I get a session. It exists à process with authority\system which doesn't reboot the computer ?

**Comment by UrielRicardo** [comment]
@STE4LTH  try migrate to simple process like notepad.exe
work for me.

**Comment by STE4LTH** [comment]
Yes, I know, but sometimes in my case I launch the script, and the computer reboot without I get session.

**Comment by mastonic** [comment]
Hello , salut, 

I have a same Issue when i have  a sesion my vm restart with a beautifull screen Bleu of wndows .

**Comment by mastonic** [comment]
Hello, 

For me i find a solution is just change the processinject i choose a process in allready in my VM Win7 for example : explorer.exe 

Pour ma part j'ai réussi à régler le pb en mettant un autre processus que celui mis par Default j'ai choisi un process qui était déjà en cours sur la VM par exemple explorer.exe
